uniffi/src/cli.rs

```diff
@@ -35,7 +35,9 @@ enum Commands {
         #[clap(long, short)]
         no_format: bool,
 
-        /// Path to the optional uniffi config file. If not provided, uniffi-bindgen will try to guess it from the UDL's file location.
+        /// Path to optional uniffi config file. This config will be merged on top of default
```

Comment:
This docstring could probably be improved - it describes the implementation but not really the UI - ie, maybe it should be described in terms of being the default config or similar? It's also worth adding some docs to https://github.com/mozilla/uniffi-rs/blob/main/docs/manual/src/bindings.md.

Reply:
> This docstring could probably be improved - it describes the implementation but not really the UI - ie, maybe it should be described in terms of being the default config or similar? It's also worth adding some docs to https://github.com/mozilla/uniffi-rs/blob/main/docs/manual/src/bindings.md.

I will update the markdown fil, but I don't agree about the description. I would agree that its quite verbose, but its not wrong. This config lets you override default config values in crate config. I'm not really sure how else to describe this config.

Reply:
Reading your comment again, I see your point. Describing it in terms of being the default config sounds very terse. But this does not correctly map to the behaviour. It does not work like the default config. It works as an override to the config in each crate.

Now I'm thinking if its worth changing the implementation to make this the "default" config for all crates, instead of overriding.
- If the config acts as a default, then what you see in a crate's config config is what you get in the bindings. This sounds convenient when thinking from individual crates perspective. Setting `foo: foo` in crate config, I can be sure that this is what I will always get.
- If the config acts as an override, then what you see in the override config is what you get in all crate's bindings. This sounds convenient when thinking from bindings generation perspective. Setting `foo: bar` in override config, I can be sure that all crates will share the same value, regardless of their config.

I'm still leaning towards the override option. Thinking about a theoretical option `force_snake_case` for Kotlin (not sure why anyone would want that, but hey), I want a way to ensure that all my crates use the same configuration value, regardless of their crate configuration. I don't want to have to go and grep all crate config files, to ensure that none of the crates using this configuration key.

I've reworded the docstring, and updated documentation file. Let me know what you think.